debezium-testing/debezium-testing-system/pom.xml

```diff
@@ -293,6 +291,48 @@
   </dependencies>
 
   <profiles>
+    <profile>
+      <id>apicurioV1</id>
+      <activation>
+        <property>
+          <name>apicurioV1</name>
```

Comment:
@jpechane  as much as I don't like this, I don't see a better way to do it. The reason for these profiles is that APv1 and APv2 need a different module dependency. However profile activation does not work with maven properties so it would essentially make the ``apicurio.crd.version`` needlessly mandatory.

Reply:
there actually is a way but it requires changes to the model generator project which I would rather avoid doing